lib/terraforming/resource/route53_zone.rb

```diff
@@ -49,7 +49,7 @@ def tfstate
       private
 
       def hosted_zones
-        @client.list_hosted_zones.hosted_zones.map { |hosted_zone| @client.get_hosted_zone(id: hosted_zone.id) }
+        @client.list_hosted_zones.map(&:hosted_zones).flatten.map { |hosted_zone| @client.get_hosted_zone(id: hosted_zone.id) }
```

Comment:
__[rubocop]__
* Line is too long. [127/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)